tests/unit/gateways/AvailableAppointments.test.js

```diff
@@ -36,22 +36,4 @@ describe('SearchProperties', () => {
     )
     expect(result).toEqual(dummyData)
   });
-
```

Comment:
Can this and the other equivalent, deleted tests be replaced with new tests?

Reply:
we are catching any exceptions thrown in the function index with sentry so these tests are now redundant - although we'd probably might want to add some tests on those files later